postgres_composite_types/forms.py

```diff
@@ -61,21 +62,25 @@ class CompositeTypeField(forms.Field):
     }
 
     def __init__(self, *args, fields=None, model=None, **kwargs):
-        if fields is None:
-            fields = {name: field.formfield() for name, field in model._meta.fields}
-        else:
-            fields = dict(fields)
+        fields = {
+            field.name: field.formfield()
+            for field in fields or model._meta.fields
+            if field.name != DummyField.name
```

Comment:
`isinstance` ?

Reply:
I don't remember now, but IIRC that didn't work because of the way Django does all this.